scipy-1.0/paper.tex

```diff
@@ -567,14 +567,14 @@ \subsection*{SciPy Matures}
 
 By the middle to late 2000s, SciPy was starting to mature after a long phase of significant
 growth and adoption.
-The informal workshops are grown into international conferences with many
-hundreds of attendees, special issues are organized and published in a
-leading scientific magazine, and the scope of the SciPy library
-narrows, while the breadth of the ecosystem is grown and encouraged
+The informal workshops grew into international conferences with many
+hundreds of attendees. Special issues were organized and published in a
+leading scientific journal, and the scope of the SciPy library
```

Comment:
I've added one. In the context of mid to late 2000s this seems to mean Dubois / IEEE CiSE special issue citation, so using that. But I note the impact factor is only ~1.0 range & the fact that we've already cited this earlier in the text are not compelling points in favor.

Reply:
I didn't know this was referring the Dubois /IEEE CiSE thing. The citation makes that clear.
I don't see the low impact factor as a reason not to cite it. It could be a reason not to mention the fact - but we should still cite if we are going to mention it, right?